README.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
+  <img width="20%" height="20%" src="resources/icon.png">
```

Comment:
Have you tried bundling the app in production mode?
Does the image path resolve correctly when the application assets are packaged?

Reply:
idk

https://github.com/readium/readium-desktop/pull/1011/files#diff-04c6e90faac2675aa89e2176d2eec7d8L9-L11 ?